### PR TITLE
v19.1.0 release notes

### LINKED ISSUE #4541: Release notes for v19.1.0 GA
Need to summarize known limitations, features, and big changes, as we did in the v2.1 release notes: https://www.cockroachlabs.com/docs/releases/v2.1.0.html. 

Also update the docs site to treat v19.1 as stable. This means creating a v19.2 directory.

### PR DESCRIPTION
**Pull request by jseldess**
Fixes #4541.

### CODE REVIEW

**Comment by awoods187** [comment]
this isn't an enterprise feature 
```
Custom Savepoints | CockroachDB now supports custom naming of SAVEPOINTs for compatibility with ORMs and other third-party tools.
-- | --
```

**Comment by jseldess** [comment]
Thanks, @awoods187. Fixed.

File: releases/v19.1.0.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+**Troubleshooting** | Added much more guidance on [troubleshooting cluster setup](../v19.1/cluster-setup-troubleshooting.html) and [troubleshooting SQL behavior](../v19.1/query-behavior-troubleshooting.html).
+**Architecture** | Added the [Life of a Distributed Transaction](../v19.1/architecture/life-of-a-distributed-transaction.html), which details the path that a query takes through CockroachDB's architecture, starting with a SQL client and progressing all the way to RocksDB (and then back out again). Also added [Reads and Writes in CockroachDB](../v19.1/architecture/reads-and-writes-overview.html), which explains how reads and writes are affected by the replicated and distributed nature of data in CockroachDB.
+**Production Guidance** | Expanded the [Production Checklist](../v19.1/recommended-production-settings.html) with more current hardware recommendations and additional guidance on storage, file systems, and clock synch. Also added a library of common [Cluster Topology Patterns](../v19.1/cluster-topology-patterns.html).
```
Comment (rmloveland) [inline releases/v19.1.0.md]: Nit: "Synch" caught my eye while reading but it's not wrong, just the less common variant according to [Merriam-Webster](https://www.merriam-webster.com/dictionary/sync).  I'd vote "sync" personally.

File: releases/v19.1.0.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+- The normalized results of certain timestamp + duration operations involving year or month durations have been adjusted to agree with the values returned by PostgreSQL.
+
+- The `CHANGEFEED` [`experimental-avro` option](../v19.1/create-changefeed.html#options) has been renamed `experimental_avro`.
+
+- Timezone abbreviations, such as `EST`, are no longer allowed when parsing or converting to a date/time type. Previously, an abbreviation would be accepted if it were an alias for the session's timezone.
+
+- The way composite [foreign key](../v19.1/foreign-key.html) matches are evaluated has changed to match the Postgres behavior. If your schema currently uses composite keys, it may require updates, since this change may affect your foreign key constraints and cascading behavior. For more details and guidance, see [this note](v2.2.0-alpha.20190114.html#composite-foreign-key-matching).
+
+- Mutation statements like [`UPDATE`](../v19.1/update.html) and [`INSERT`](../v19.1/insert.html) no longer attempt to guarantee mutation or output ordering when an `ORDER BY` clause is present. It is now an error to use `ORDER BY` without `LIMIT` with the `UPDATE` statement.
```
Comment (rmloveland) [inline releases/v19.1.0.md]: This is also true of `DELETE`, docs in progress via https://github.com/cockroachdb/docs/issues/4378

Comment (jseldess) [reply releases/v19.1.0.md]: Added.

File: releases/v19.1.0.md (hunk 1 above)
Comment (jseldess) [reply releases/v19.1.0.md]: Good catch, but I'll just changed it to `synchronization`.

**Pull request merged**